GSoC/2016/README.md

```diff
@@ -0,0 +1,29 @@
+# Note for 2016 GSoC application
+
+It's my first time in the GSoC application. Due to my lack of the process, I missed the deadline of updating my enrollment letter, which finally caused the eligibility.
```

Comment:
It's my first time applying for the GSoC program. For lack of experiences, I missed the deadline of submitting my enrollment letter, which finally caused the disapproval.